convention-develocity-gradle-plugin/plugins/gradle-2-through-4/settings.gradle

```diff
@@ -1,7 +1,29 @@
 plugins {
-    id 'com.gradle.develocity' version '3.17.3'
+    id 'com.gradle.develocity' version '3.17.3' // replace with this convention plugin once published
```

Comment:
Not sure what you mean with this comment. There is no plan to publish this convention plugin. It serves as a template that other can copy and adjust to their needs.